### PR TITLE
Friends List Pull Reqest

### LINKED ISSUE #4: Create Friends List
Create a friends list and allow people to add other users to that list!

### PR DESCRIPTION
**Pull request by ewink1977**
The #4 friends list functionality has been completed from a basic level. I wanted to go ahead and push this up now so the Front End folks can see the logic behind it. 

I think we can provide more functionality by creating a more detailed bio/profile page. This can be delayed till later, or I can continue to build on what we have so far (since we have our own User class, we don't need to build a separate UserProfile, and can just build off that).

### CODE REVIEW

**Comment by ewink1977** [comment]
> Don't merge this commit till everyone has the .env file, otherwise, the application won't run. As well, the library python-dotenv needs to be installed as well.

I removed the dotenv stuff. **The SK and APIs are still exposed**, but since this doesn't have any important information as of yet, it's no big deal.

File: myWeek/settings.py [hunk 1]
```diff
@@ -20,7 +24,8 @@
 # See https://docs.djangoproject.com/en/3.1/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = '2t=62lhwk6&3!(kdva-++dd=28%ukl1m(t=599$ji27t83*dts'
+# ADDED THIS BACK IN FOR SIMPLICITY AND WHO CARES IF IT'S STOLE, RIGHT? /ew
```
Comment (nheyland) [inline myWeek/settings.py:27]: true lol

File: myWeek/wsgi.py [hunk 2]
```diff
@@ -8,6 +8,8 @@
 """
 
 import os
+from dotenv import load_dotenv
+load_dotenv()
 
 from django.core.wsgi import get_wsgi_application
```
Comment (nheyland) [inline myWeek/wsgi.py:15]: Did you create this file? // did you put this on a server somewhere? Just curious if this is in preparation to make this live

**Review by nheyland — APPROVED** [review]
Looks good to me! Dont think anything will affect what I have in my branch, feel free to merge when  @jarthurj confirms this won't interfere with anything hes got goin on.

File: myWeek/wsgi.py (hunk 2 above)
Comment (ewink1977) [reply myWeek/wsgi.py:15]: Sorry, that's a library that can be installed with `pip (or pipenv) install python-dotenv`. It's to make sure the .env file loads. It's unnecessary up to this point but when/if we add things like go live and want to conceal the secret key, use an S3 bucket, hide some APIs or other sensitive info (like email credentials when the invitation function is done), dotenv will be necessary.

**Review by jarthurj — APPROVED** [review]

**Pull request merged**